sql/mysql/sqlspec.go

```diff
@@ -307,8 +352,10 @@ func binarySpec(t *schema.BinaryType) (*sqlspec.Column, error) {
 func stringSpec(t *schema.StringType) (*sqlspec.Column, error) {
 	switch t.T {
 	case tVarchar, tMediumText, tLongText, tTinyText, tText, tChar:
-		s := strconv.Itoa(t.Size)
-		return specutil.NewCol("", "string", specutil.LitAttr("size", s)), nil
+		if t.Size == 0 {
+			return &sqlspec.Column{Type: t.T}, nil
+		}
+		return specutil.NewCol("", "string", specutil.LitAttr("size", strconv.Itoa(t.Size))), nil
```

Comment:
A tiny patch to support text fields. Will remove after we decide to tackle virtual fields.